serde-gcp-cloudevents/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
```

Comment:
This whole file could be replaced with: 


```kotlin
plugins {
    id("io.micronaut.build.internal.module")
}
dependencies {
    annotationProcessor(projects.serdeProcessor)
    implementation(projects.serdeJackson)
    api(libs.managed.google.cloudevent.types)
}
micronautBuild {
    binaryCompatibility {
        enabled.set(false)
    }
}
//TODO remove with Micronaut 4
configurations.named("testRuntimeClasspath") {
    resolutionStrategy.dependencySubstitution {
        substitute(module("io.micronaut:micronaut-jackson-databind"))
            .using(project(":serde-jackson")).because("we want to test Micronaut without jackson databind")
    }
}

```